src/components/Homepage/home.css

```diff
@@ -0,0 +1,378 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+.container{
+	max-width: 1376px;
+	height: 786px;
+	cursor: pointer;
+	position: relative;
+}
+
+/* .container-wrapper{
+	border: 5px solid darkgoldenrod;
+	margin: 0 30px;
+} */ 
+
+.sixty {
+	width: 115px;
+	height: 48px;
+	left: 60px;
+	top: 15px;
+	font-family: 'Gruppo', cursive;
+	font-weight: normal; 
+	font-style: normal;
+	font-size: 36px;
+	line-height: 35px;
+	color: #000;
+	position: absolute;
+
+}
+
+.sixty a {
+color: #333;
+}
+
+.sixty a:hover{
+	color: #333;
+}
+
+.ellipse{
+	width: 12px;
+	height: 12px;
+	left: 102px;
+	top: 19px;
+	box-sizing: border-box;
+	position: absolute;
+}
+
+.login-btn{
+	width: 35px;
+	height: 23px;
+	top:12px;
+	font-weight: normal;
+	font-style: normal;
+	font-size: 14px;
+	font-family: 'Roboto', sans-serif;
+	line-height: 161.3%;
+	position: absolute;
+	left: 21px;
+	color: #fff;
+}
+
+.signUp-btn{
+	width: 48px;
+	height: 23px;
+	position: absolute;
+	top:12px;
+	left: 20px;
+	font-weight: normal;
+	font-style: normal;
+	font-size: 14px;
+	font-family: 'Roboto', sans-serif;
+	line-height: 161.3%;
+
+}
+
+#login{
+	position: absolute;
+	width: 83px;
+	height: 44px;
+	left: 1169px;
+	top: 14px;
+	background-color: #507EF5;
+	box-shadow: 0px 1px 4px rgba(0,0,0,0.1);
+	border-radius: 30px 0px 0px 30px;
+	color: #fff;
+	box-sizing: border-box;
+}
+
+#sign{
+	position: absolute;
+	width: 96px;
+	height: 44px;
+	left: 1250px;
+	top: 14px;
+	background-color: #356DFB;
+	box-shadow: 0px 1px 4px rgba(0,0,0,0.1);
+	border-radius: 0px 30px 30px 0px;
+	color: #fff;
+	box-sizing: border-box;
+}
+
+#sign:hover, #login:hover, #get-started-btn:hover{
+		background-color: rgba(0, 0, 173, 0.753);
+}
+
+a {
+	text-decoration: none;
+	color: #fff;
+}
+
+a:hover {
+	color: #000;
+}
+
+.first-section-text {
+	position: absolute;
+	width: 538px;
+	height: 325px;
+	left: 60px;
+	top: 107px;
+	font-weight: 600;
+	font-size: 48px;
+	font-family: 'Open Sans', sans-serif;
+	line-height: 65px;
+}
+
+.first-cards-box {
```

Comment:
Think you need to format this entire file. 
Your editor should have a formatter